rocket_sqlite_oracle/src/db.rs

```diff
@@ -17,6 +17,7 @@ pub struct Signatures(sqlx::SqlitePool);
 pub struct USDTickerResponse {
     pub timestamp: u64,
     pub price: u64,
+    pub units: String,
```

Comment:
In the context of this response, I think this should be singular because there is only one price.